pkg/operation/botanist/kubeapiserver.go

```diff
@@ -348,6 +351,9 @@ func (b *Botanist) computeKubeAPIServerServiceAccountConfig(ctx context.Context,
 		out.Issuer = *config.ServiceAccountConfig.Issuer
 	}
 	out.AcceptedIssuers = config.ServiceAccountConfig.AcceptedIssuers
+	if out.Issuer != defaultIssuer && !utils.ValueExists(defaultIssuer, out.AcceptedIssuers) {
```

Comment:
If issuer and accepted issuer both are not provided in that case what should be the accepted issuers? in the current scenario it is being set to nil, but shouldn't be default issuer should be added to the accepted issuer?

Reply:
In this case, the `kube-apiserver` will be configured with `--service-account-issuer=<default-issuer>` only which makes it issue tokens with the `<default-issuer>` and verify them with the `<default-issuer>` as well. No need to specify it twice.